### DISCUSSION TITLE
Can I deploy a turborepo as a single project to vercel?

### DISCUSSION THREAD

**Discussion by devdev-dev** [vercel/turborepo#159] [Help] [answered]
I'm working with Turboepo the first time and it worked like a charm locally. Out-of-the-box, Smooth and fast.  
Is it possible to deploy the whole repository with all of its apps and packages as one project to Vercel? 
As for now, I created a project for each "app" in my monorepo. But then I have to manage URL rewrites or multi-zone. Is there any alternative?

Thanks in Advance!

**Comment by jaredpalmer** [comment, marked as answer]
Thanks for posting. Great question. 

Not at the moment. Each app needs to be deployed individually as its own project. We know this takes too long and are actively working on improving this along with better supporting monorepo codebases on the platform in general.

**Reply by devdev-dev** [reply]
Thanks for answering me directly Jared. I'm looking forward to future enhancements regarding this topic. 

I really enjoy your work!

**Reply by LinnJS** [reply]
@jaredpalmer has this changed? I don't think it has but in my opinion the fact that the docs show `app` when the app directory is `apps` it is very confusing and to me lends its self to looking like a "yes you can deploy all apps on vercel from a single project" I think a more clear example project would be `nextjs` so its not one char off of the shared `apps` dir. 

I just went to reconfigure my project to be one project then found out its not a feature yet which led me to this old post.

[Docs](https://vercel.com/docs/concepts/monorepos/turborepo)

![image](https://user-images.githubusercontent.com/23741323/211667264-a68f1c02-1344-4d30-b6cd-dd4bad74a1b6.png)
